src/classes/CON_Preferred_TEST.cls

```diff
@@ -54,12 +54,19 @@ public with sharing class CON_Preferred_TEST {
 		insert contacts;
 		Test.stopTest();
 		
-		contacts = [select Email, Phone, UniversityEmail__c, WorkEmail__c, AlternateEmail__c, HomePhone, WorkPhone__c, OtherPhone, MobilePhone 
-					from Contact where ID in :contacts order by LastName];
-		contact1 = contacts[0];
-		contact2 = contacts[1];
-		contact3 = contacts[2];
-		contact4 = contacts[3];
+		contacts = [SELECT Email, LastName, Phone, UniversityEmail__c, WorkEmail__c, AlternateEmail__c, HomePhone, WorkPhone__c, OtherPhone, MobilePhone 
```

Comment:
This looks a bit hardcoded but I guess it was like this before. We do have a utility class to create test contacts. Not a big deal thou. Just fyi :)

Reply:
Yes its VERY hard-coded but to me that's kind of the point. In a unit test, we need to know exactly the data we are woking with. The original issue here was that we were ordering by last name and the last name's were named such that doing an order by gave us a guarantee that element 0 was the first ordered name, etc. However, we can't use the lastname field in an order by if Shield is enabled. This means we couldn't necessarily say that the 0 element was the record we wanted. 
By creating a map by name we can then retrieve the exact records we want to compare. It feels very hard-coded but in a unit test, I would argue that it should be very brittle. Overall, I feel we should never rely on the order in which a query result is handed back to us for testing data. This was a way to ensure we knew exactly what record we are working with, without having to rely on result being returned in a specific order.